src/util/prolog/prolog.test.ts

```diff
@@ -122,3 +123,27 @@ describe("PrologResult.getResultArray()", () => {
 			expect(results).toStrictEqual([["grandmotherm"], ["grandmotherf"], ["false"]]);
 		});
 });
+
+describe("PrologSession.importFile()", () => {
+	test('Imports the family.pl File Correctly"', async () => {
```

Comment:
It'd be nice to describe what case you're testing; so instead of just specifying the file you're trying to use, explain what behavior you're testing by using that file - in this case e.g. `can import a file that doesn't import any other modules`, in the next one `recursively imports the contents of modules used in a file`; not sure what you're testing in the third case but I guess you get the idea :D